package.json

```diff
@@ -14,7 +14,7 @@
     "@changesets/cli": "^2.27.1",
     "@playwright/test": "^1.41.1",
     "@types/node": "^20.11.11",
-    "prettier": "^2.7.1",
+    "prettier": "^3.2.5",
```

Comment:
Update `prettier` for [TS 5.0 support](https://github.com/prettier/prettier/issues/14240).